build.gradle.kts

```diff
@@ -546,15 +480,15 @@ afterEvaluate {
 	}
 
 	fun getTaskForPublish(): TaskProvider<out DefaultTask> {
```

Comment:
Given the new behaviour of `compressJar`, the output of `compressJar` should probably always be used for publishing now over `shadowJar`.